gulpfile.js

```diff
@@ -1,52 +1,50 @@
-'use strict';
-
 const gulp = require('gulp');
 const watch = require('gulp-watch');
 const sass = require('gulp-sass');
 const autoprefixer = require('gulp-autoprefixer');
 const rename = require('gulp-rename');
 const sourcemaps = require('gulp-sourcemaps');
-const clean = require('del');
-
-process.env.NODE_ENV = 'development';
+const del = require('del');
+const { resolve } = require('path')
```

Comment:
Resolve makes easier to concatenate paths. If you'll set `ROOT` to `./directory`, that will fail because you have to add one more slash at the end. But with this package, you don't care. It resolves paths for us.